client/packages/system/src/Program/ProgramSearchInput.tsx

```diff
@@ -0,0 +1,49 @@
+import { Autocomplete } from '@openmsupply-client/common';
+import { ProgramFragment } from '@openmsupply-client/programs';
+import React from 'react';
+import { FC } from 'react';
+
+interface ProgramSearchInputProps {
```

Comment:
Pretty much a copy of "ClinicianSearchInput". The only difference is I did the query (to get the program list) in the Toolbar (parent) rather than in here, as I wanted to be able to not show the program selector if there are no programs.